README.md

```diff
@@ -55,3 +55,13 @@ docker build -t jupyter/demo .
 docker build -t jupyter/minimal common/
 ```
 
+#### `jupyter/minimal`
+You will need to run this code snippet before building the docker image
+```
+git clone https://github.com/ericdill/scikit-xray-examples
```

Comment:
What the `prepare_for_docker.py` script does is download and unzip a couple
of files that are needed for the demos, so I would imagine that this clone
(and more importantly download/unzip) step should be done once, not every
build?

On Mon, Aug 17, 2015, 7:06 PM Dan Allan <notifications@github.com> wrote:

> In README.md
> <https://github.com/danielballan/docker-demo-images/pull/3#discussion_r37247779>
> :
>
> > @@ -55,3 +55,13 @@ docker build -t jupyter/demo .
> >  docker build -t jupyter/minimal common/
> >  ```
> >
> > +#### `jupyter/minimal`
> > +You will need to run this code snippet before building the docker image
> > +```
> > +git clone https://github.com/ericdill/scikit-xray-examples
>
> Take a look at line 38 in Dockerfile. We can do the cloning as part of the
> docker build process -- no need for manual setup.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/danielballan/docker-demo-images/pull/3/files#r37247779>
> .
>